maintainers/maintainer-list.nix

```diff
@@ -10686,6 +10686,16 @@
     github = "pulsation";
     githubId = 1838397;
   };
+  zseri = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add zseri``.